bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/util/ReferenceCountedHashMapTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Created by j-clap on 10/17/19. */
+public class ReferenceCountedHashMapTest {
+
+  private final String[] KEYS = {"first", "second"};
+  private final Integer[] VALUES = {1, 2};
+
+  @Test
+  public void test_insertion_works() {
```

Comment:
We follow the camelcase naming convention for test case names for this repository. Could you please update these names accordingly.

Reply:
I'm just following the same conventions as the other tests in Utils.